src/solver/variable/surveyresults/reportbuilder.hxx

```diff
@@ -213,8 +213,7 @@ public:
     static void Run(const ListType& list, SurveyResults& results, unsigned int numSpace = 9999)
     {
         // Standard - Not related to anything
-        if (CDataLevel & Category::standard)
```

Comment:
Leave ?

Reply:
RunStandard was used only if mode == adq draft (see file below) so I deleted it